### DISCUSSION TITLE
Postal is not sending emails - email status is pending

### DISCUSSION THREAD

**Discussion by speedeonic** [postalserver/postal#1726] [Help with using Postal] [answered]
Hello,

I am new to Postal mail server and I need help on this problem or bug. 
It may also be helpful for others who face this problem.

I have installed Postal on Ubuntu 18.04 (64 Bit) and Docker.
I followed the instructions by a video tutorial on YouTube and installed everything like Docker compose, Maria Db, RabbitMQ etc. I setup all the required dns and finally Postal got installed and it seemed working.

But when I tried to send emails from newly installed Postal, emails didn't deliver and the email status is "Pending".

I checked the Postal status and found these values.

postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Exit 1
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Exit 1

Can anyone please help me and guide me on how to fix it?

Thank you
Faiz

**Comment by willpower232** [comment, marked as answer]
You will need to have a look at the logs of the exited containers to see what has happened that caused them to stop working.

**Reply by speedeonic** [reply]
Actually, I am not a programmer, I just followed a video tutorial on YouTube and article which provided all the commands to use.

Can you please share the commands to check the logs of the exited containers?

Thank you

**Reply by willpower232** [reply]
I think you can do `postal logs worker` to see the docker output.

**Comment by Chira3124** [comment]
I think you should double check the correct information in the file /opt/postal/config/postal.yml , 3 parts below.
main_db:
message_db:
rabbitmq:
